### PR TITLE
Bug 1143230 - Columns not aligned consistently on app detail pages

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Columns are now aligned consistently
fixed

### CODE REVIEW

File: src/media/css/detail/more-info.styl [hunk 1]
```diff
@@ -5,10 +5,13 @@
...
 
 .app-more-metadata {
+        margin: 0 -12px;
     dl {
-        display: inline-block;
+        display: block;
         vertical-align: top;
-        width-calc('50% - 2px');
+        width: 50%;
+        float: left;
```
Comment (mstriemer) [inline src/media/css/detail/more-info.styl]: If you leave this as `display: inline-block;` you shouldn't need to set it to `float: left;`.

File: src/media/css/detail/more-info.styl [hunk 2]
```diff
@@ -5,10 +5,13 @@
 @import '../lib';
 
 .app-more-metadata {
+        margin: 0 -12px;
```
Comment (mstriemer) [inline src/media/css/detail/more-info.styl]: This should be indented by 4 spaces so it line up with the `dl` below. Please add a blank line after also.

Comment (mstriemer) [inline src/media/css/detail/more-info.styl]: I think you'll want to set `font-size: 0;` here also.

**Comment by mstriemer** [comment]
r+wc ping me again when you've made the changes and I'll merge. Thanks!

**Comment by atiqueahmedziad** [comment]
@mstriemer Please check it now, updated :)

File: src/media/css/detail/more-info.styl [hunk 3]
```diff
@@ -5,10 +5,14 @@
 @import '../lib';
 
 .app-more-metadata {
+    margin: 0 -12px;
```
Comment (mstriemer) [inline src/media/css/detail/more-info.styl]: We normally sort our CSS properties alphabetically. So `font-size` should come before `margin`. `padding` is out of place in the `dl` styles also. Sorry I should have mentioned this before.

**Comment by mstriemer** [comment]
Looks good! If you want to update the ordering of the styles that'd be great. It isn't a big deal otherwise though so I'll merge this later.

**Comment by atiqueahmedziad** [comment]
Thanks ...updated :) 
learning things...will not do this mistakes in future :)

**Comment by mstriemer** [comment]
A bit of an arbitrary decision on our part but it ends up being helpful in the end.

Thanks!

**Pull request merged**